hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/endpoint/VersionEndpointTask.java

```diff
@@ -75,6 +76,10 @@ public EndpointStateMachine.EndPointStates call() throws Exception {
           // If end point is passive, datanode does not need to check volumes.
           String scmId = response.getValue(OzoneConsts.SCM_ID);
           String clusterId = response.getValue(OzoneConsts.CLUSTER_ID);
+          DatanodeLayoutStorage layoutStorage
+              = new DatanodeLayoutStorage(configuration, clusterId);
```

Comment:
Can you keep the dataNodeId as null if it's not easy to get the datanode UUID in this class?

Reply:
Thanks for the review @ChenSammi. Handled it, Can you please take a look.